applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_eigensolver.py

```diff
@@ -33,7 +33,8 @@ def GetDefaultSettings(cls):
                 "max_iteration"         : 1000,
                 "tolerance"             : 1e-6,
                 "number_of_eigenvalues" : 5,
-                "echo_level"            : 1
+                "echo_level"            : 1,
+                "compute_modal_decomposition": false
```

Comment:
```suggestion
            "compute_modal_decomposition": false
```
this way you don't have to mess with the eigensolver settings

Reply:
done